webapp/actions/post_actions.jsx

```diff
@@ -363,6 +363,15 @@ export function createPost(post, doLoadPost, success, error) {
     );
 }
 
+export function updatePost(post, success) {
+    Client.updatePost(
+        post,
+        success,
```

Comment:
I wouldn't worry about squashing the commits, but can you add the `loadPosts` call in here? It should look similar to this https://github.com/mattermost/platform/blob/master/webapp/actions/channel_actions.jsx#L194